src/Frontend/Pages/CreateLobby.tsx

```diff
@@ -85,42 +77,12 @@ export function CreateLobby({ match }: RouteComponentProps<{ contract: string }>
         .getConstants()
         .then((config) => {
           setStartingConfig({
-            // Explicitly defaulting this to false
+            ...config,
             WHITELIST_ENABLED: false,
-            // TODO: Figure out if we should expose this from contract
-            START_PAUSED: false,
-            ADMIN_CAN_ADD_PLANETS: config.ADMIN_CAN_ADD_PLANETS,
-            WORLD_RADIUS_LOCKED: config.WORLD_RADIUS_LOCKED,
-            WORLD_RADIUS_MIN: config.WORLD_RADIUS_MIN,
-            DISABLE_ZK_CHECKS: config.DISABLE_ZK_CHECKS,
-            PLANETHASH_KEY: config.PLANETHASH_KEY,
-            SPACETYPE_KEY: config.SPACETYPE_KEY,
-            BIOMEBASE_KEY: config.BIOMEBASE_KEY,
-            PERLIN_MIRROR_X: config.PERLIN_MIRROR_X,
-            PERLIN_MIRROR_Y: config.PERLIN_MIRROR_Y,
-            PERLIN_LENGTH_SCALE: config.PERLIN_LENGTH_SCALE,
-            MAX_NATURAL_PLANET_LEVEL: config.MAX_NATURAL_PLANET_LEVEL,
-            TIME_FACTOR_HUNDREDTHS: config.TIME_FACTOR_HUNDREDTHS,
-            PERLIN_THRESHOLD_1: config.PERLIN_THRESHOLD_1,
-            PERLIN_THRESHOLD_2: config.PERLIN_THRESHOLD_2,
-            PERLIN_THRESHOLD_3: config.PERLIN_THRESHOLD_3,
-            INIT_PERLIN_MIN: config.INIT_PERLIN_MIN,
-            INIT_PERLIN_MAX: config.INIT_PERLIN_MAX,
-            SPAWN_RIM_AREA: config.SPAWN_RIM_AREA,
-            BIOME_THRESHOLD_1: config.BIOME_THRESHOLD_1,
-            BIOME_THRESHOLD_2: config.BIOME_THRESHOLD_2,
-            PLANET_LEVEL_THRESHOLDS: config.PLANET_LEVEL_THRESHOLDS,
-            PLANET_RARITY: config.PLANET_RARITY,
-            LOCATION_REVEAL_COOLDOWN: config.LOCATION_REVEAL_COOLDOWN,
-            // TODO: Implement when we add this scoring contract back
+            START_PAUSED: true,
             CLAIM_PLANET_COOLDOWN: 0,
-            // TODO: Need to think through this implementation a bit more, even if only toggling planet types
-            PLANET_TYPE_WEIGHTS: config.PLANET_TYPE_WEIGHTS,
-            // TODO: Rename in one of the places
-            // TODO: Implement... Needs a datetime input component (WIP)
-            TOKEN_MINT_END_TIMESTAMP: 1948939200, // new Date("2031-10-05T04:00:00.000Z").getTime() / 1000,
-            PHOTOID_ACTIVATION_DELAY: config.PHOTOID_ACTIVATION_DELAY,
-            SILVER_SCORE_VALUE: config.SILVER_SCORE_VALUE,
+            ADMIN_PLANETS: [],
+            TOKEN_MINT_END_TIMESTAMP: Date.now() + 21600000, // six hours from now
```

Comment:
BAD - if someone makes a lobby but doesn't start it right away, this will fail. We should have a token mint end timestamp for into the future.

Reply:
also this messes with the lobby config, we need to think through how that will work

Reply:
What is `this` and what is `messing with` ?